src/sentry/static/sentry/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -0,0 +1,416 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import keyBy from 'lodash/keyBy';
+
+import {Organization, Integration, IntegrationProvider, RouterProps} from 'app/types';
+import {RequestOptions} from 'app/api';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {Hooks} from 'app/types/hooks';
+import {t} from 'app/locale';
+import {trackIntegrationEvent} from 'app/utils/integrationUtil';
+import AsyncComponent from 'app/components/asyncComponent';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import space from 'app/styles/space';
+import AddIntegrationButton from 'app/views/organizationIntegrations/addIntegrationButton';
+import Access from 'app/components/acl/access';
+import Tag from 'app/views/settings/components/tag';
+import Button from 'app/components/button';
+import Alert from 'app/components/alert';
+import Tooltip from 'app/components/tooltip';
+import InlineSvg from 'app/components/inlineSvg';
+import ExternalLink from 'app/components/links/externalLink';
+import InstalledIntegration, {
+  Props as InstalledIntegrationProps,
+} from 'app/views/organizationIntegrations/installedIntegration';
+import marked, {singleLineRenderer} from 'app/utils/marked';
+import HookStore from 'app/stores/hookStore';
+import withOrganization from 'app/utils/withOrganization';
+import {growDown, highlight} from 'app/styles/animations';
+import {sortArray} from 'app/utils';
+
+type State = {
+  configurations: Integration[];
+  provider: {information: IntegrationProvider};
+  tab: string;
+  newlyInstalledIntegrationId: string;
+};
+
+type Props = RouterProps & {
```

Comment:
@NisanthanNanthakumar I think I added `RouterProps` before I knew how to deal with the router properly. I should have removed this earlier. We should use `RouteComponentProps` instead of this. See `sentryApplicationDetails.tsx` as an example.